README.md

```diff
@@ -6,42 +6,59 @@
 
 ## Bridge Overview
 
-The POA Bridge allows users to transfer assets between two chains in the Ethereum ecosystem. It is composed of several elements which are located in different POA Network repositories:
+The POA Bridge allows users to transfer assets between two chains in the Ethereum ecosystem. 
+
+It is composed of several elements which are located in different POA Network repositories:
 
 **Bridge Elements**
+
 1. An oracle written in NodeJS, contained in this repository.
-2. [Solidity smart contracts](https://github.com/poanetwork/poa-bridge-contracts). Used to manage bridge validators, collect signatures, and confirm asset relay and disposal.
-3. [Bridge UI Application](https://github.com/poanetwork/bridge-ui). A DApp interface to transfer tokens and coins between chains.
-4. [Bridge Monitor](https://github.com/poanetwork/bridge-monitor). A tool for checking balances and unprocessed events in bridged networks.
-5. [Bridge Deployment Playbooks](https://github.com/poanetwork/deployment-bridge). Manages configuration instructions for remote deployments.
+2. [Solidity smart contracts](https://github.com/ColuLocalNetwork/poa-bridge-contracts). Used to manage bridge validators, collect signatures, and confirm asset relay and disposal.
```

Comment:
We need to update references from `ColuLocalNetwork` to `poanetwork`. One option is to update it as part of this PR. Another option is to update them on a new PR after merging this one to avoid introducing this references on `ColuLocalNetwork:master`

Reply:
I think it's best to do it in a separate PR. We will do this after poanetwork/poa-bridge-contracts#140 and this PR will be merged.